src/backend/src/prisma/schema.prisma

```diff
@@ -182,7 +182,7 @@ model User {
   deletedMilestones               Milestone[]               @relation(name: "milestoneDeleter")
   receivedAnnouncements           Announcement[]            @relation(name: "ReceivedAnnouncements")
   createdAnnouncements            Announcement[]            @relation(name: "CreatedAnnouncements")
```

Comment:
I dont know why everyone is capitalizing the relation names. All relation names should be camelcase